src/v2/Components/__stories__/Editorial.story.tsx

```diff
@@ -74,15 +73,14 @@ export const TextCentered = () => {
   )
 }
 
+// TODO: Do we need a proper sticky library?
 export const TextFixedLeftScrollRight = () => {
   return (
     <Page title="<TextFixedLeftScrollRight />">
       <GridColumns>
-        <StickyContainer>
```

Comment:
FWIW; the immediate child of `<GridColumns>` always has to be a `<Column>` — I need to make the typing more restrictive so that this will type error.